omod/src/main/resources/webModuleApplicationContext.xml

```diff
@@ -24,9 +24,9 @@
 
 	 
 	<!-- Annotation based controllers -->
-	<bean class="org.springframework.web.servlet.mvc.annotation.DefaultAnnotationHandlerMapping"/>
+	<bean class="org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerMapping"/>
```

Comment:
is this required? Component scan should wire things up by default? I think it will work without any explicit mapping as well.  In other modules, we have removed this bean def .e.g in appointments which is used in OMRS 2.5 as well - https://github.com/Bahmni/openmrs-module-appointments/commit/0a257ecf4c1ab0e01cced7ec33d4a9a5dddf681a

Reply:
Cleared the xml based bean creation. It is not required, the integration tests passed after removing it.
Thanks for pointing this out.